erpnext/projects/report/profitability/profitability.json

```
@@ -0,0 +1,39 @@
+{
+ "add_total_row": 0,
+ "columns": [],
+ "creation": "2021-03-18 10:19:40.124932",
+ "disable_prepared_report": 0,
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "filters": [],
+ "idx": 0,
+ "is_standard": "Yes",
+ "json": "{}",
+ "modified": "2021-03-18 10:20:15.559305",
+ "modified_by": "Administrator",
+ "module": "Projects",
+ "name": "Profitability",
+ "owner": "Administrator",
+ "prepared_report": 0,
+ "ref_doctype": "Timesheet",
+ "report_name": "Profitability",
```

```suggestion
 "report_name": "Project Profitability",
```

1. Rename report to Project Profitability since it is with respect to customer-wise timesheets, etc.
Update wherever necessary.
2. Also add a Project column and filter.